composer.json

```diff
@@ -17,11 +17,6 @@
             "SilverStripe\\HTML5\\Tests\\": "tests"
         }
     },
-    "extra": {
```

Comment:
Why remove this alias? By doing this, you'll only be able to install this branch with the "dev-master" constraint.

Reply:
yes. However, I'm not sure what's wrong with "dev-master" though? Just had a discussion with @NightJar yesterday that aliases aren't really making it more clear than just following our conventions (https://docs.silverstripe.org/en/4/contributing/code/#picking-the-right-version) and go for dev-master?

Reply:
Yeah I don't agree with Dylan here sorry. This repo aside, I'd love to be able to `composer install silverstripe/installer 5.x-dev`

Reply:
Yeah, though, I'm not patching the core modules at the moment. Here I'm removing the to be incorrect branch alias, since the master should become next major, but adding and maintaining the branch aliases is not worth it (imho), just adds extra maintenance burden.
Another point is the aliases aren't part of our branch naming conventions at the moment, so I'm not sure "I like it" is valuable enough to have it here.

Reply:
To be honest @dnsl48 I think you're changing things that is just making more work for us. If you're changing this `composer.json` file for the next major then you should adjust the alias to `3.x-dev`. But as you're converting `master` to be "the next major" you also need to branch a `2` branch to accept contributions for the next minor. This is just creating more branches for maintenance. In my experience (and opinion) it's much easier just to wait until somebody has a major change before converting `master` to be "the next major".

Reply:
To be clear, updating the travis.yml config is super helpful and that'd be great to merge, but maybe let's not worry about changing branch "targets" right now?

Reply:
Yes, there's plan to create the branch `2` when the master is upgraded.
The main idea is to follow the same branch naming conventions across core and cwp modules and recipes.
I'm not really sure we are discussing a real problem about master - there's no plan to release the next major for this module, so there shouldn't be much need in using alias for that.

Reply:
@ScopeyNZ sorry, I guess I didn't give you much context in the description (just because I'm doing this for lots of modules due to https://github.com/silverstripe/cwp-recipe-kitchen-sink/issues/46#issuecomment-547734409)

The Idea is to create major branches for all modules and make master to be the next major, so it all follows the same conventions as the core modules and we don't have to remember what's cwp and what's core.

Reply:
Yeah I can see that. I just commented there. The issue reads "should have a branch targetting the next minor" - which I'm saying they all do. I think this rule can apply to core too - they're just more likely to have major changes pending release.

Reply:
I guess it's also worth saying that the "core" modules are the only modules that still have this idea of lockstep for minor/major releases - which already has them in a special category.